frontend-react/src/components/shared/Table/Table.module.scss

```diff
@@ -0,0 +1,53 @@
+@use 'uswds-core' as *;
+
+.reportstreamTable {
```

Comment:
Using camel case is pretty conventional in css modules because we'd go from `className={styles.reportstreamTable}` to `className={styles['reportstreamTable']}`. Just want to confirm, is that what we want to do?

Reply:
Will change to use name of component as container class, ie `styles.Table`